LogMonitor/src/LogMonitor/etw-monitor-docs/README.md

```diff
@@ -0,0 +1,72 @@
+# ETW Monitor Documentation
+
+To monitor ETW logs using Log Monitor you required either a valid provider name or a provider GUID. Specifying either the provider name or provider GUID or both in the json configuration will work.
+
+The provider that you pass in the json config file provider must register with ETW and instrumented to generate manifest-based events. 
```

Comment:
I think this should read `must be registered` as `must register` may be taken as the provider registers itself (e.g. and it needs LogMonitor run w/ privileges to allow that).